### PR TITLE
Davidroossien wt okta 343763

### PR DESCRIPTION
**Pull request by davidroossien-okta**
Note: I had to rename the folder to include underlines, so there are deleted files from contractor-expiry-notice, and only the added files inside contractor_expiry_notice should be reviewed.

### CODE REVIEW

File: workflows/contractor_expiry_notification/readme.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Many organizations utilize contractors as well as full time employees. A contractor would usually have a contract expiry date. That is the date at which their current contract is due to expire. Certain people within the organization, like the contractor's manager, will need to be notified ahead of the expiry date, so they can potentially renew the employees contract.
+
+This template contains workflows which read a custom attribute on the user's Okta profile. This custom attribute holds the user's current contract expiry date. Based on a set number of days, a future date is calculated. The workflows compile a list of all users that have a matching contract expiry date and sends emails to a pre-configured list of recipients.
+
+These workflows also demonstrate how static configuration values, like days, timezone and email recipients, can be externalized into a workflows table and read at runtime. This reduces the need to make modifications to the workflows, such that changes only involve updating the respective configuration table.
+
+
+## Workflow Summary
+
+Below is a summary of the flows included in the flowpac:
```
Comment (charleschan-okta) [inline workflows/contractor_expiry_notification/readme.md:15]: typo: flowpac should be flowpack

Comment (charleschan-okta) [inline workflows/contractor_expiry_notification/readme.md:15]: typo: flowpac should be flopack

File: workflows/contractor_expiry_notification/workflow.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "office365mail"
+  ],
+  "links": [
+    {
+      "title": "Video Walk-through",
+      "destination": "https://www.youtube.com/watch?v=huNrhJWEqUA"
+    },
+    {
+      "title": "Sample Data",
+      "destination": "https://github.com/okta/workflows-templates/blob/master/workflows/contractor-expiry-notification/configurationTableData.csv"
```
Comment (charleschan-okta) [inline workflows/contractor_expiry_notification/workflow.json:16]: The path is referring to .../contractor-expiry-notification/..., should be .../contractor_expiry_notification/... (ie. using underscore)

File: workflows/contractor_expiry_notification/workflow.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    }, 
+    {
+      "title": "Setup Documentation",
+      "destination": "https://github.com/okta/workflows-templates/blob/master/workflows/contractor-expiry-notification/readme.md"
```
Comment (charleschan-okta) [inline workflows/contractor_expiry_notification/workflow.json:20]: The path is referring to .../contractor-expiry-notification/readme.md, should be .../contractor_expiry_notification/readme.md (ie. using underscore)

**Review by charleschan-okta — CHANGES_REQUESTED** [review]
1) workflow.json file needs to be updated (using underscore instead of dash for the path)
2) minor typo in readme file
3) u probably want to remove all the files that are still using dash in the path to avoid confusion. If these are merged, they can also show up in the catalog

**Review by charleschan-okta — APPROVED** [review]
1) workflow.json file needs to be updated (using underscore instead of dash for the path)
2) minor typo in readme file
3) u probably want to remove all the files that are still using dash in the path to avoid confusion. If these are merged, they can also show up in the catalog

**Comment by davidroossien-okta** [comment]
closed because we merged a different PR

**Pull request closed without merging**